frontend/src/components/AppFooter.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <footer id="footer">
-        <h2>Copyright &copy; SmartCourse 2018</h2>
+        <p>Copyright &copy; SmartCourse 2018</p>
```

Comment:
thank god

Reply:
The change I deliver, but not the one you deserve @NunoDasNeves